.github/workflows/cicd.yml

```diff
@@ -536,9 +536,9 @@ jobs:
     uses: ./.github/workflows/reusable-test.yml
     with:
       name: km_performance
-      pre_test: .\setup_ebpf_cicd_tests.ps1
+      pre_test: .\setup_ebpf_cicd_tests.ps1 -KmTracing $true
```

Comment:
I see this parameter is true in all cases - is it needed, then?

Reply:
Before you renamed the Tracing to KmTracing, the parameter was set to true by Install-eBPFComponentsOnVM 
`Install-eBPFComponents -Tracing $true -KMDFVerifier $true`
IMO no need to bubble it up here. You can delete it from the setup/cleanup PS1 scripts.